src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -599,6 +599,10 @@ private String startBulkCaseBookPDFJob(String studyOid,
 			//Setting the destination file
 	        String fullFinalFilePathName = this.getMergedPDFcasebookFileName(studyOid, participantId);
 	        int index= fullFinalFilePathName.lastIndexOf("\\");
+	        //LINUX path
```

Comment:
use File.separator instead of this if/else condition

Reply:
done